### ISSUE TITLE
No DNS zone creation

### ISSUE THREAD

**Issue by thegr33nhornet** [blesta/module-ispconfig#1]
As the title states, no DNS zone creation.

v1.1 also not showing php/ssh chroot options?

**Comment by JReissmueller** [comment]
So this is a feature request then?  You would like to include a service action whereby a client may create a DNS zone for themselves?

**Comment by thegr33nhornet** [comment]
No, I would like the module to create a DNS zone using an ispconfig template for the site automatically, also an email domain ideally.
